### ISSUE TITLE
Many queries are not conversational.

### ISSUE THREAD

**Issue by betterclever** [fossasia/susi_skill_data#115]
**Actual Behaviour**

Many queries in wiki currently are not conversational. For example, " wiki * ", " current time in * ", " distance between * and * "

**Expected Behaviour**

From my point of view, queries should be in form of a question as well like , "search wikipedia for *", "what is the current time in *" , "what is the distance between * and *".
The interaction should be more natural and questions asked to assistants are in from of questions most of the times.

**Steps to reproduce it**

N/A

**LogCat for the issue**

N/A

**Screenshots of the issue**

N/A

**Would you like to work on the issue?**

This should be solved collectively. I will collaborate too.

**Comment by hpdang** [comment]
@betterclever we need both questions and 'non-conversational' phrases as you describe " wiki * ", " current time in * ", " distance between * and * ", we need to include them all.  @saurabhjn76 is working on those at the moment.

**Comment by betterclever** [comment]
Yes, exactly. I wrote the same in issue description that we need queries in form of question "as well". It was not about converting queries to question form but including both and that is clear from the issue. 
I faced difficulty when I was making YouTube video of Susi interaction headlessly on Raspberry Pi, since most queries couldn't be used due to this reason. " Susi! wiki dog " , doesn't seem natural thus I raised the issue :smile:
